runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -277,11 +278,11 @@ private ClueScroll findClueScroll()
 		if (clueScrollText != null)
 		{
 			// Remove line breaks and also the rare occasion where there are double line breaks
-			String text = clueScrollText.getText()
+			String text = Text.removeTags(clueScrollText.getText()
```

Comment:
`removeTags` already removes tags, so you don't need to remove them again.

Reply:
Ok. But then the `"-<br>", "-"` replacement is not necessary?

Reply:
True replacement of this is not necessary

Reply:
@devLotto would probably know best, but I think it will create double spaces if its just replacing `<br>` with ` `

Reply:
If ```"TEXT-<br>TEXT" ``` would be replaced by ```"<br>", " "``` the result would couse whitespaces 
```"TEXT- TEXT"``` instead of no-spaces ```"TEXT-TEXT"```